cleanlab/experimental/datalab/TestDatalab.py

```diff
@@ -0,0 +1,113 @@
+# Copyright (C) 2017-2024  Cleanlab Inc.
+# This file is part of cleanlab.
+#
+# cleanlab is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# cleanlab is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with cleanlab.  If not, see <https://www.gnu.org/licenses/>.
+"""
+The experimental feature that construct a datalab instance with the statistic information from a trained datalab.
+"""
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any, Dict, List, Optional, Union
+
+import pandas as pd
+import numpy as np
+
+from cleanlab.datalab.datalab import Datalab
+from cleanlab.datalab.internal.issue_finder import IssueFinder
+
+if TYPE_CHECKING:  # pragma: no cover
+    import numpy.typing as npt
+    from datasets.arrow_dataset import Dataset
+    from scipy.sparse import csr_matrix
+
+    DatasetLike = Union[Dataset, pd.DataFrame, Dict[str, Any], List[Dict[str, Any]], str]
+
+
+class TestDatalab(Datalab):
+    def __init__(
+        self,
+        trained_datalab: Datalab,
+        data: "DatasetLike",
+        task: str = "classification",
+        label_name: Optional[str] = None,
+        image_key: Optional[str] = None,
+        verbosity: int = 1,
+    ):
+        super().__init__(data, task, label_name, image_key, verbosity)
+        self.trained_datalab = trained_datalab
```

Comment:
let's call this: self.datalab

Reply:
and when writing about this object, let's say is:

"The Datalab object fitted to your original training dataset"